### PR TITLE
[pt-br] Add docs/reference/using-api/health-checks.md

### PR DESCRIPTION
**Pull request by DaviMatheus**
fixes the issue that looks like it was abandoned: Translate health-checks.md to pt-br #36015

### CODE REVIEW

File: content/pt-br/docs/reference/using-api/health-checks.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: Kubernetes API health endpoints
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+O Kubernetes {{< glossary_tooltip term_id="kube-apiserver" text="servidor de API" >}} fornece endpoints de API para indicar o status atual do servidor de API. Esta página descreve esses endpoints da API e explica como você pode usá-los.
```
Comment (femrtnz) [inline content/pt-br/docs/reference/using-api/health-checks.md:8]:
```suggestion
O {{< glossary_tooltip term_id="kube-apiserver" text="servidor de API" >}} do Kubernetes fornece endpoints de API para indicar o status atual do servidor de API. Esta página descreve esses endpoints da API e explica como você pode usá-los.
```

File: content/pt-br/docs/reference/using-api/health-checks.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+O endpoint `livez` pode ser usado com a [flag](/docs/reference/command-line-tools-reference/kube-apiserver) `--livez-grace-period` para especificar a duração de inicialização.
+Para um desligamento elegante, você pode especificar a [flag](/docs/reference/command-line-tools-reference/kube-apiserver) `--shutdown-delay-duration` junto com o endpoint `/readyz`.
+As máquinas que verificam o `healthz`/`livez`/`readyz` do servidor API devem se basear no código de status HTTP.
+Um código de status `200` indica que o servidor API está `healthy`/`live`/`ready`, dependendo do endpoint chamado.
+As opções mais detalhadas mostradas abaixo destinam-se a ser usadas manualmente para depurar seu cluster ou entender o estado do servidor de API.
+
+Os exemplos seguintes mostrarão  como você pode interagir com os endpoints de API health.
+
+Para todos os endpoints, você pode usar o parâmetro `verbose` para imprimir as verificações e seus status.
+Isso pode ser útil para um operador humano depurar o status atual do servidor de API, ele não se destina a ser consumido por uma máquina:
```
Comment (femrtnz) [inline content/pt-br/docs/reference/using-api/health-checks.md:25]: Not sure if this makes sense in porptuguese. Maybe change it to
```suggestion
Isso pode ser útil para um usuário depurar o status atual do servidor de API, ele não se destina a ser consumido por uma máquina:
```

File: content/pt-br/docs/reference/using-api/health-checks.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    [+]poststarthook/start-cluster-authentication-info-controller ok
+    [+]poststarthook/start-kube-aggregator-informers ok
+    [+]poststarthook/apiservice-registration-controller ok
+    [+]poststarthook/apiservice-status-available-controller ok
+    [+]poststarthook/kube-apiserver-autoregistration ok
+    [+]autoregister-completion ok
+    [+]poststarthook/apiservice-openapi-controller ok
+    healthz check passed
+
+O Kubernetes servidor de API também suporta excluir verificações específicas.
```
Comment (femrtnz) [inline content/pt-br/docs/reference/using-api/health-checks.md:59]:
```suggestion
O servidor de API do Kubernetes também suporta excluir verificações específicas. 
```

File: content/pt-br/docs/reference/using-api/health-checks.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    healthz check passed
+
+## Verificações individuais de health
+
+{{< feature-state state="alpha" >}}
+
+Cada verificação individual de health expõe um endpoint HTTP e pode ser verificado individualmente.
+O esquema para as verificações de health individuais é `/livez/<healthcheck-name>` onde  `livez` e `readyz` são usados para indicar se você quer verificar a vivacidade ou a prontidão do servidor de API.
+O caminho `<healthcheck-name>` pode ser descoberto usando a flag `verbose` demonstrada acima, e pegue o caminho entre `[+]` e `ok`.
+Essas verificações individuais de health, não devem ser consumidas por máquinas mas podem ser úteis para  sererm depuradas manualmente em um sistema:
```
Comment (femrtnz) [inline content/pt-br/docs/reference/using-api/health-checks.md:96]:
```suggestion
Essas verificações individuais de health não devem ser consumidas por máquinas mas podem ser úteis para  serem depuradas manualmente em um sistema:
```

**Review by femrtnz — COMMENTED** [review]
Thanks, this looks great! Just added some small comments but lgtm

**Comment by stormqueen1990** [comment]
Hello there, @DaviMatheus! :wave: 

Thank you very much for your work on this PR. However, it has been open and reviewed for a while with no action taken. For this reason, I will be closing this PR. If you wish to continue your work on this, please feel free to reopen it.

Should you have any questions, please don't hesitate to reach out to the `#kubernetes-docs-pt` channel in Kubernetes Slack.

---

Olá, @DaviMatheus! :wave:

Obrigada pelo teu trabalho neste PR. Devido a inatividade após a revisão, irei fechá-lo. No entanto, caso desejes continuar teu trabalho, podes reabrir o PR a qualquer momento.

Se tiveres dúvidas, não hesita em contatar-nos através do canal `#kubernetes-docs-pt` no Slack do Kubernetes.

/close

**Pull request closed without merging**